backend/booking_engine/README.md

```diff
@@ -0,0 +1,47 @@
+![Chronogolf][crest]
+
+# The Booking Engine challenge
+
+Each golf club manages players reservation through our system.
+When a player wants to make a reservation, the request needs to be passed through a booking engine that will determine if the request is valid. The system will then confirm the reservation if valid.
+
+Your mission consists of building a simple booking engine and returns the list of confirmed reservations.
```

Comment:
`Your mission consists of building a simple booking engine that validates reservations, and returns the list of confirmed reservations.`